debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (native)
```

Comment:
Should be `quilt`, I believe, `native` is only for tools written by Debian themselves